src/Mailgun/Api/Pagination.php

```diff
@@ -22,12 +22,52 @@
     abstract protected function safeDeserialize(ResponseInterface $response, $className);
 
     /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationNext($response)
+    {
+        return $this->getPaginationUrl($response->getNextUrl(), get_class($response));
+    }
+
+    /**
+     * @param mixed $response
+     *
+     * @return mixed|null
+     */
+    public function getPaginationPrevious($response)
```

Comment:
Or even this ^^.

I like just being able to do `$resp->previous()` and so on, as it's very clean and not redundant.